src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -217,8 +217,10 @@ private void UpdateIndicator(Size bounds)
                     TemplateProperties.Container2AnimationEndPosition = barIndicatorWidth2 * 1.66; // Position at 166%
 
                     // Remove these properties when we switch to fluent as default and removed the old one.
```

Comment:
I think we aren't removing the default theme. So we should probably somehow adapt it to the new properties instead.

Reply:
@maxkatz6 @danwalmsley